index.html

```diff
@@ -13,11 +13,54 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:ital,wght@0,400;0,500;1,100&family=Fredoka+One&family=Open+Sans+Condensed:ital,wght@1,300&family=Outfit:wght@200&family=PT+Sans+Caption:wght@700&family=Roboto:wght@300&display=swap"
+    rel="stylesheet">
 </head>
 <body>
-  <header>
+  <header class="container nav-bar-heading">
```

Comment:
You should put the background color on the `header` element, but make a child of that element with the `container` class, so that the background color on the header fills the full width of the viewport, but the content will be constrained to the same width as the rest of the content on the page.